core/src/main/java/org/apache/calcite/sql/validate/SqlConformanceEnum.java

```diff
@@ -77,11 +77,33 @@
 
   /** Conformance value that instructs Calcite to use SQL semantics
    * consistent with Microsoft SQL Server version 2008. */
-  SQL_SERVER_2008;
+  SQL_SERVER_2008,
 
+  /** Conformance value that instructs Calcite to use SQL semantics
```

Comment:
You might be missing BIG_QUERY here?

Reply:
BIG_QUERY should be referenced on line 49 in this file. For the other comment, I can make a parent "DialectQuidemTest" to remove some of the copied code